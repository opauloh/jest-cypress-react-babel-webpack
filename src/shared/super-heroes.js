const heroes = [
  {
    name: 'Dynaguy',
    powers: ['laser', 'fly']
  },
  {
    name: 'Paulo',
    powers: ['press arms']
  }
]
export const getFlyingSuperHeroes = () => heroes.filter(hero => hero.powers.includes('fly'))