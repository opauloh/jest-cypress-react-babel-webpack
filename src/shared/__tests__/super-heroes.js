import { getFlyingSuperHeroes} from '../super-heroes';

test('return super heroes that can fly', () => {
  const flyingHeroes = getFlyingSuperHeroes()
  expect(flyingHeroes).toMatchSnapshot()
})