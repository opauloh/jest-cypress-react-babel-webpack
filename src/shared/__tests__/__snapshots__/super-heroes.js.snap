// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`return super heroes that can fly 1`] = `
Array [
  Object {
    "name": "Dynaguy",
    "powers": Array [
      "laser",
      "fly",
    ],
  },
]
`;
